datadriven.go

```diff
@@ -496,12 +495,10 @@ func (td TestData) Fatalf(tb testing.TB, format string, args ...interface{}) {
 	tb.Fatalf("%s: %s", td.Pos, fmt.Sprintf(format, args...))
 }
 
+var blankLineRe = regexp.MustCompile(`(?m)^[\t ]*\n`)
```

Comment:
we can use the more typical $ instead of \n no?

Reply:
Unfortunately, no (that was my first attempt).
Go does something very weird at the end of the input. I added a comment to explain.